### ISSUE TITLE
Errors in generate-json-crud.sql

### ISSUE THREAD

**Issue by VicSmith** [Microsoft/sql-server-samples#252]
Running Code Gen Returns following:

Can anyone assist?
SQL Server Samples\sql-server-samples\samples\features\json\code-generator\generate-json-crud.sql

Msg 208, Level 16, State 1, Procedure GetTableColumns, Line 45 [Batch Start Line 56]
Invalid object name 'STRING_SPLIT'.
Msg 208, Level 16, State 1, Procedure GetPkColumns, Line 5 [Batch Start Line 108]
Invalid object name 'codegen.GetTableColumns'.
Msg 208, Level 16, State 1, Procedure GetTableDefinition, Line 5 [Batch Start Line 119]
Invalid object name 'codegen.GetTableColumns'.

**Comment by VicSmith** [comment]
Found the problem.
My compatibility level on the database in Azure was set to to SQL Server 2008 (100).

Actually to figure this out I figured the database was too old.  It has  been around in Azure for years.
So, I created a brand new DB Server in Azure and a brand New Database and when setting up in SS Management Studio, I found the setting.... 
Note for others...
In SS Management Studio
Right click DB> Select Page > Options > Compatibility Level > SQL Server vNext(140)...

BTW... this JSON SUPPORT IS FREAKIN AWESOME!  Also, thanks for creating these generation scripts.
![image](https://user-images.githubusercontent.com/1848257/27030434-d6101c7a-4f31-11e7-8b85-aec07c940d5e.png)
